server/models/Profile.js

```diff
@@ -0,0 +1,45 @@
+const mongoose = require('mongoose');
+
+const profileSchema = new mongoose.Schema({
```

Comment:
I think it would also be good to establish a linkage between the User model and the corresponding Profile. Maybe a reference in the User model or in both models, think through how you would need to query it to decide one way or the other 😄